### PR TITLE
refactor(kubernetes): kubernetes is now a module

### PR DESCRIPTION
**Pull request by kevinawoo**
This converts kubernetes so that it can be built as a module. I followed the conventions laid out in `docker`, `google`, and `amazon` modules.

Here's an included script to help test it out locally.
https://gist.github.com/kevinawoo/1fb8db8d54918ad05dc2b328481fd77c

to run it it: 
```
curl -O https://gist.githubusercontent.com/kevinawoo/1fb8db8d54918ad05dc2b328481fd77c/raw/26edbadec7dceda1aecddb375111c12cd3084dc2/test-kubernetes-module.sh
chmod +x test-kubernetes-module.sh
./test-kubernetes-module.sh
```

### CODE REVIEW

**Comment by kevinawoo** [comment]
cc @danielpeach can you take a look too?

File: app/scripts/modules/kubernetes/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      NODE_MODULE_PATH,
+      path.resolve('.'),
+    ],
+    alias: {
+      '@spinnaker/kubernetes': path.join(__dirname, 'src'),
+      'coreImports': path.resolve(basePath, 'app', 'scripts', 'modules', 'core', 'src', 'presentation', 'less', 'imports', 'commonImports.less'),
+      'kubernetes': path.join(__dirname, 'src')
+    }
+  },
+  watch:  process.env.WATCH === 'true',
```
Comment (danielpeach) [inline app/scripts/modules/kubernetes/webpack.config.js]: Extra space (`watch: process`)

File: app/scripts/modules/kubernetes/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        use: [
+          { loader: 'cache-loader' },
+          { loader: 'thread-loader', options: { workers: 3 } },
+          { loader: 'babel-loader' },
+          { loader: 'envify-loader' },
+          { loader: 'eslint-loader' } ,
```
Comment (danielpeach) [inline app/scripts/modules/kubernetes/webpack.config.js]: Extra space between closing brace and comma.

**Comment by kevinawoo** [comment]
Fixed the whitespace issues ⬜️ +🔥  =  ✅

**Comment by jrsquared** [comment]
Makes sense to me.

**Pull request merged**